### PR TITLE
Api#67 expose participant UUID field

### PR DESCRIPTION
**Pull request by jinjagit**
Trying to link up my fork to this PR, after GitHub weirdness

### CODE REVIEW

**Comment by jinjagit** [comment]
GitHub has been fixed. This is now ready to merge.

File: priv/repo/migrations/20200710182224_create_results.exs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+defmodule LiquidVoting.Repo.Migrations.CreateResults do
+  use Ecto.Migration
+
+  def change do
+    create table(:results) do
+      add :in_favor, :integer, default: 0
+      add :against, :integer, default: 0
+      add :proposal_url, :string, default: false, null: false
+      add :organization_uuid, :uuid
```
Comment (niklaslong) [inline priv/repo/migrations/20200710182224_create_results.exs:9]: I'm wondering if we should change this to `organisation_id` already (and everywhere else)?

File: lib/liquid_voting_web/schema/schema.ex [hunk 2]
```diff
@@ -19,9 +19,9 @@ defmodule LiquidVotingWeb.Schema.Schema do
       resolve(&Resolvers.Voting.participants/3)
     end
 
-    @desc "Get a participant by its id"
+    @desc "Get a participant by its uuid"
```
Comment (niklaslong) [inline lib/liquid_voting_web/schema/schema.ex:22]: Should this be `id`? (I've noticed both `uuid` and `id` being used (also in tests))

Comment (jinjagit) [reply lib/liquid_voting_web/schema/schema.ex:22]: I created a new issue for this, yesterday.

File: priv/repo/migrations/20200710182224_create_results.exs (hunk 1 above)
Comment (jinjagit) [reply priv/repo/migrations/20200710182224_create_results.exs:9]: I created a new issue for this, yesterday.

File: lib/liquid_voting_web/schema/schema.ex (hunk 2 above)
Comment (jinjagit) [reply lib/liquid_voting_web/schema/schema.ex:22]: Where in tests (other than organization_uuid)? I've just been through all the tests again, and can't see any.

File: priv/repo/migrations/20200710182224_create_results.exs (hunk 1 above)
Comment (jinjagit) [reply priv/repo/migrations/20200710182224_create_results.exs]: This is now done, and related new issue closed.

File: lib/liquid_voting_web/schema/schema.ex (hunk 2 above)
Comment (jinjagit) [reply lib/liquid_voting_web/schema/schema.ex]: OK, found it. There was one in voting_results_test.ex.

**Pull request merged**